MapboxDirections/MBRouteLeg.swift

```diff
@@ -34,15 +38,18 @@ open class RouteLeg: NSObject, NSSecureCoding {
             if let speed = jsonAttributes["speed"] {
                 segmentSpeeds = speed as? [CLLocationSpeed] ?? []
             }
-            if let nodes = jsonAttributes["nodes"] {
-                openStreetMapNodeIdentifiers = nodes as? [Int64] ?? []
+            if let congestion = jsonAttributes["congestion"] as? [String] {
+                congestionLevels = congestion.map {
+                    CongestionLevel(description: $0)!
```

Comment:
> congestion values are tied to the Mapbox Traffic v1 tile set in some fashion

In theory yes, but practically they don't share code. Right now they share values, but in the future it is possible that either introduces a value that the other one doesn't copy (e.g. a `closed` tag makes sense for Traffic tiles but in Mapbox Directions we'll never route you through a closed street, thus a `closed` congestion annotation will never occur.